### ISSUE TITLE
Move MYCHECK_RCPT to cnf file

### ISSUE THREAD

**Issue by ottok** [ottok/mariadb-10.0#4]
```
11:46 #maria: < Jb_boin> otto, it would be interresting to move the MYCHECK_RCPT variable from the 
                         debian-start script to a /etc/default/mysql (or mariadb) ; its used in 
                         debian-start.inc.sh as the mail recipient if crashed tables are detected by the 
                         check_for_crashed_tables() function
```

**Comment by jb-boin** [comment]
Can be closed, it has been fixed on pull #27.

**Comment by ottok** [comment]
Thanks @jb-boin for your contributions and follow-up. I appreciate it.
